_tests/travis-checks

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# Execute Markdown Linter
+mdl README.md _posts _pages --style _linter/markdown-linter-style.rb
+
+# Github Pages needs the baseurl field to be "m-lab.github.io" to render properly in forked 
+# repos, but when we deploy to production, we need the baseurl field to be empty. This 
+# command replaces "m-lab.github.io" with an empty string.
+sed -e '/^baseurl/s/\/m-lab\.github\.io//' _config.yml > _config-deploy.yml
+
+# sets --disable-external flag for HTMLProofer if quick argument exists
+quick="${1/%quick/disable-external}"
```

Comment:
OK, switched this to an if statement now.  I was going back and forth about whether or not it would best to separate that into a function now since that logic is becoming longer.  I kept it simple for now, but if you think additional logic could eventually go in here then maybe we consider a function for this?  Maybe that's overkill...

Implemented an `if` statement now though at least in https://github.com/m-lab/m-lab.github.io/pull/81/commits/eabd7618abc1915efa0d31b50e1c9b29e56e6078